spec/jobs/application_job_spec.rb

```diff
@@ -17,15 +17,6 @@ def perform(param_a, param_b, should_raise)
 end
 
 RSpec.describe ApplicationJob do
-  include ActiveJob::TestHelper
-
-  around do |example|
-    old_adapter = ActiveJob::Base.queue_adapter
-    ActiveJob::Base.queue_adapter = :sidekiq
-    example.run
-    ActiveJob::Base.queue_adapter = old_adapter
-  end
```

Comment:
I don't think we really need to be testing Sidekiq. Best to keep sidekiq out of the test suite.

Reply:
yeah, agree, ideally